src/test/java/Homework19.java

```diff
@@ -0,0 +1,25 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest  {
+
+    @Test
+    public void Login() {
+
+        login("anna.stonaieva@testpro.io", "te$tStudent");
+    }
+        public void openPlaylist() throws InterruptedException {
+            WebElement Playlist = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+            Playlist.click();
+            Thread.sleep(1000);
+        }
+
+        public void deletePlaylist() throws InterruptedException {
+        WebElement deletePlaylistButton = driver.findElement(By.cssSelector(".btn-delete-playlist"));
+        deletePlaylistButton.click();
+        Thread.sleep(1000);
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Reply:
Ok. We didn`t learn this split yet. but I will use it in my next test.